roles/robottelo/tasks/main.yml

```diff
@@ -47,18 +49,24 @@
   lineinfile:
     dest: "{{ robottelo_directory }}/robottelo.properties"
     regexp: "^hostname="
-    line: "hostname={{ ansible_fqdn }}"
+    line: "hostname={{ robottelo_hostname }}"
     state: present
     backrefs: yes
 
-- name: Set robottelo logging
+- name: Set robottelo SSH
   lineinfile:
```

Comment:
Any reason this can't use `replace` here in as well like in logging below? Same for the above hostname.

Reply:
No reason it can't use replace. I tend to think of lineinfile as change one single line vs. replace which is designed to replace all occurrences. If you prefer use of replace I can update.